packages/hubble-sdk/src/Hubble.ts

```diff
@@ -75,23 +75,59 @@ export class Hubble {
     return stability;
   }
 
+  private borrowingMarketStateToDecimals(state: BorrowingMarketState, pubkey: PublicKey) {
+    state = replaceBigNumberWithDecimal(state);
+    state.depositedCollateral.amounts = replaceBigNumberWithDecimal(state.depositedCollateral.amounts);
+    state.inactiveCollateral.amounts = replaceBigNumberWithDecimal(state.inactiveCollateral.amounts);
+    state.collateralRewardPerToken = replaceBigNumberWithDecimal(state.collateralRewardPerToken);
```

Comment:
maybe also stablecoinRewardPerToken?

Reply:
Collateral reward per token is an array so it has to be replaced like this, stablecoinRewardPerToken is a BN and that gets converted by this line already: `state = replaceBigNumberWithDecimal(state);`

The `replaceBigNumberWithDecimal` method works for Object.entries, but unfortunately it doesn't automatically convert arrays/nested types.